### PR TITLE
Created new contact UI

### LINKED ISSUE #8: Create a new contact UI

### PR DESCRIPTION
**Pull request by anil-anand**
Fixes #8 
Also fixes the suggested changes in #11.

@yedhink _a
Please review.

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Contacts/Table.jsx
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/Table.jsx]: I am not able to understand why you are doing an assignment while passing the arguments. Please put up a video and explain to me your thought process. I don't need a text reply. A video is must.

**Review by yedhink — CHANGES_REQUESTED** [review]
@anil-anand

File: app/javascript/src/components/Dashboard/Contacts/Table.jsx
Comment (anil-anand) [reply app/javascript/src/components/Dashboard/Contacts/Table.jsx]: Video: https://www.loom.com/share/d42c30056bee4ddcb1a7dddb89b577f8

Comment (yedhink) [reply app/javascript/src/components/Dashboard/Contacts/Table.jsx]: @anil-anand About positional args, please read: https://academy.bigbinary.com/learn-react/react-best-practices/structuring-code#:~:text=Positional%20parameters%20vs%20Named%20parameters

About the MenuBar code, watch: https://www.loom.com/share/aaf8c41dc3864949bea0a3026ac6158a

Comment (anil-anand) [reply app/javascript/src/components/Dashboard/Contacts/Table.jsx]: Yedhin, I don't think this is a mistake.
In the code, there is a `Container` below the `MenuBar`, which holds an `Open MenuBar` / `Close MenuBar` button. 

#### Code:

image

#### Button in the UI:

image

When I deleted the `Container` from the copy-pasted code, I forgot to delete the `div`. Sorry for the trouble!

And thank you for the link regarding positional parameters and named parameters!

Comment (yedhink) [reply app/javascript/src/components/Dashboard/Contacts/Table.jsx]: >Yedhin, I don't think this is a mistake.

Got it, In that case we don't need to raise any issue.

Comment (anil-anand) [reply app/javascript/src/components/Dashboard/Contacts/Table.jsx]: Okay. After making the above changes to the file, can I proceed with deploying the application?

Comment (yedhink) [reply app/javascript/src/components/Dashboard/Contacts/Table.jsx]: Yes.

Comment (yedhink) [reply app/javascript/src/components/Dashboard/Contacts/Table.jsx]: You don't have to wait for my approval to merge PRs and continue with next tasks.

**Pull request merged**
